### DISCUSSION TITLE
How can Android and iOS dynamically and in real time pass parameters to a game? Also, how can the game call methods from Android or iOS?

### DISCUSSION THREAD

**Discussion by zhanjunjie2019** [hajimehoshi/ebiten#3018] [Q&A] [answered]
How can Android and iOS dynamically and in real time pass parameters to a game? Also, how can the game call methods from Android or iOS?

**Comment by hajimehoshi** [comment, marked as answer]
You can expose Go APIs to the Android/iOS world, and vice versa. You can use the same way as `gobind`. See https://pkg.go.dev/golang.org/x/mobile/cmd/gobind

EDIT: [Reverse binding](https://pkg.go.dev/golang.org/x/mobile/cmd/gobind#hdr-Reverse_bindings) are not available.
